src/salmon/index/test.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e
+
+dir_in="$meta_resources_dir/test_data"
+
+echo ">>> Run salmon_index"
+"$meta_executable" \
+  --transcripts $dir_in/transcriptome.fasta \
+  --index index
+
+echo ">>> Checking whether output exists"
+[ ! -d "index" ] && echo "'index' does not exist!" && exit 1
+[ -z "$(ls -A 'index')" ] && echo "'index' is empty!" && exit 1
```

Comment:
Could you include some checks on whether the output is correct and not just empty?

Reply:
Added a check to verify whether the `info.json` file exists. This file should be generated if the run was successful.